configure

```diff
@@ -1101,6 +1112,10 @@ def configure_intl(o):
         pprint.pformat(icu_config, indent=2) + '\n')
   return  # end of configure_intl
 
+def configure_engine(o):
+  engine = options.engine or 'v8'
```

Comment:
@pritambaral  , @benjamingr  - Default is v8 engine as seen in [vcbuild.bat](https://github.com/Microsoft/node/commit/12cc5d95f143bc45f5a6a832a0335e9a8f9d152e#diff-f1d059c92a388e475daa1fbc2a88aa3aR38)